src/dtgtk/paint.c

```diff
@@ -198,6 +198,56 @@ void dtgtk_cairo_paint_solid_arrow(cairo_t *cr, gint x, int y, gint w, gint h, g
   FINISH
 }
 
+void dtgtk_cairo_paint_sortby_alt(cairo_t *cr, gint x, gint y, gint w, gint h, gint flags, void *data)
```

Comment:
I wonder if it wouldn't be better to have only 1 icon and to just enclose the 'right part" of the icon under a `if(flags & CPF_DIRECTION_UP)....else...` that would simplify code (here and in the filter bar) and avoid the duplication of the arrow part...

Reply:
Done. Indeed, better code. Thanks